ParisBNF/abb/BNFabb39.xml

```diff
@@ -368,28 +369,28 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
  <msItem xml:id="p1_i1.4.25">
       <locus from="54vb" to="55vb"/>
             <title type="complete">Figures of speech from the New Testament</title>
-           <textLang mainLang="gez" otherLangs="am"></textLang>
+           <textLang mainLang="gez" otherLangs="am"/>
     <incipit xml:lang="gez"><hi rend="rubric">እጽሕፍ፡</hi> ነገረ፡ አምሳል፡ ዘሐዲስ፡ ሕግ፡
   </incipit>
         </msItem>
           </msItem>
+                      
```

Comment:
add `</msItem>` here to close p1_i1 (wrong nesting!)

Reply:
If I add `</msItem>` the system reports a mistake.

Reply:
Of course, you need to remove the one in line 735

Reply:
The nesting is wrong, see the image above. you should close` p1_i1` (after the four sub-parts) before going on to `p1_i2`

So you need to add one MORE closing tag `/msItem` before `p1_i2` (now it remained the same, you need to add in line 375), and remove one extra one after` p1_i9` in line 378, as Oxygen will prompt you to